corehq/motech/dhis2/dhis2_config.py

```diff
@@ -26,7 +26,7 @@ class FormDataValueMap(DocumentSchema):
 
 
 class Dhis2FormConfig(DocumentSchema):
-    xmlns = StringProperty()
+    xmlns = StringProperty(required=True)
```

Comment:
Do all the current form configs have this property? I think this may cause errors loading the doc if some of them on prod do not have this already specified

Reply:
Yes, all the ones on prod do. (I would have been surprised if it was ever missing, because if "xmlns" is not set, the configuration doesn't match a form and never gets used.)